theforeman.org/scripts/debian/add_changelog.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -xe
+
+if [[ $gitrelease == true ]] || [[ -n $pr_number ]]; then
+
+  echo "--Adding changelog entry"
+
+  PACKAGE_NAME=$(head -n1 debian/changelog|awk '{print $1}')
+  DATE=$(date -R)
+  RELEASE="9999-${VERSION}-${os}+scratchbuild+${BUILD_TIMESTAMP}"
```

Comment:
`os` is unset for the plugins case, which might generate "funny" versions here, but we could also just do `os=plugins` in `setup_sources_plugin.sh` which will accidentally produce exactly the right version string ;)

Reply:
I think this is set right? https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/yaml/jobs/packaging_build_deb_plugin.yaml#L13

Reply:
oh, right, it is, but it's "wrong", as the plugins aren't specific to bionic.

okay, so maybe the best approach would be: for core and deps, we set `SUITE=${os}` (like we do in `stage_pacakges`), and for plugins we set `SUITE=plugins` and use `${SUITE}` not `${os}` here?